rest-api/src/main/java/com/cloudbees/workflow/util/ServeJson.java

```diff
@@ -26,9 +26,13 @@
         @Override
         public Object invoke(StaplerRequest request, StaplerResponse response, Object instance, Object[] arguments)
                 throws IllegalAccessException, InvocationTargetException {
-            Object o = target.invoke(request, response, instance, arguments);
-
-            return new JsonResponse(JSONReadWrite.jsonMapper,o);
+            try {
+                Object o = target.invoke(request, response, instance, arguments);
+                return new JsonResponse(JSONReadWrite.jsonMapper,o);
+            } catch (Exception e) {
+                // TODO: Can be removed and ServletException added to throws declarations from 1.651+
+                throw new RuntimeException("Unexpected exception while serving JSON", e);
```

Comment:
This is here because `invoke` declaration in `Interceptor` class throws `ServletException` from 1.651 onwards, so build fails for the 2.x line (as well as for any version from and including 1.651)